utils/src/types.rs

```diff
@@ -0,0 +1,40 @@
+use crate::diff_score;
```

Comment:
the utils crate is a bit cursed, we wanted to phase this out for quite some time now (should be doable now), just because it's not really structured.

imo the `common` crate is the successor.

so I'd suggest we move this to common/src/abi/types.rs